build/configs/rtl8720e/hello/defconfig

```diff
@@ -1207,7 +1207,7 @@ CONFIG_EXAMPLES_RDP=y
 # CONFIG_EXAMPLES_TLS_SELFTEST is not set
 # CONFIG_EXAMPLES_TLS_SERVER is not set
 CONFIG_EXAMPLES_WIFIMANAGER_TEST=y
-CONFIG_WIFIMANAGER_TEST_TRIAL=y
+CONFIG_WIFIMANAGER_TEST_TRIAL=5
```

Comment:
If you change the config type from y/n to int, you should change the type from Kconfig also.

Reply:
Dear @jeongchanKim ,
The original CONFIG_WIFIMANAGER_TEST_TRIAL=y was a mistake, the WIFIMANAGER_TEST_TRIAL is default int type in Kconfig file. Thanks for reminding.